### PR TITLE
Add documentation for minio running in KinD

### LINKED ISSUE #1436: Add instructions for minio with KinD
Due to https://github.com/kubernetes-sigs/kind/issues/99, using a `nodePort` service for Minio will not work with KinD unless the user is running on Linux.

We need to update the Minio docs to mention `kubectl port-forward` when paired with KinD

### PR DESCRIPTION
**Pull request by nrb**
Fixes #1436

Signed-off-by: Nolan Brubaker <brubakern@vmware.com>

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -230,6 +230,26 @@ You must also get the Minio URL, which you can then specify as the value of the
 
 1.  Edit your `BackupStorageLocation` YAML, adding `publicUrl: <URL_FROM_PREVIOUS_STEP>` as a field under `spec.config`. You must include the `http://` or `https://` prefix.
 
+## Expose Minio outside your cluster with Kubernetes in Docker (KinD):
+
+Kubernetes in Docker currently does not have support for NodePort services (see [this issue](https://github.com/kubernetes-sigs/kind/issues/99)). Instead, you can use a port-forward to access the Minio bucket.
```
Comment (carlisia) [inline docs/get-started.md]: s/Instead/In this case

File: docs/get-started.md [hunk 2]
```diff
@@ -230,6 +230,26 @@ You must also get the Minio URL, which you can then specify as the value of the
...
+
+In one terminal, run the following:
```
Comment (carlisia) [inline docs/get-started.md]: s/In one/In a

File: docs/get-started.md [hunk 3]
```diff
@@ -230,6 +230,26 @@ You must also get the Minio URL, which you can then specify as the value of the
...
+
+```shell
+MINIO_POD=$(kubectl get pods -n velero -l component=minio -o jsonpath='{.items[0].metadata.name}')
```
Comment (carlisia) [inline docs/get-started.md]: Where does the `items` array come from?

Comment (nrb) [reply docs/get-started.md]: `kubectl get pods` returns a `List`, which an `items` field including all matching resources, even if there is only one. In this particular case, there should only ever be one if the instructions were followed, hence hardcoding the first index.

Comment (nrb) [reply docs/get-started.md]: Here's the Go definition of the `List` type: https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L887

Comment (carlisia) [reply docs/get-started.md]: Thanks for that, will check.

I'm getting this:

```
bash-3.2$ kubectl get po
NAME                                READY   STATUS    RESTARTS   AGE
nginx-deployment-84bbbd6dc7-st589   2/2     Running   0          44h
bash-3.2$ MINIO_POD=$(kubectl get pods -n velero -l component=minio -o jsonpath='{.items[0].metadata.name}')
error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}```

Comment (nrb) [reply docs/get-started.md]: Looks like you didn't have a minio pod; do you have anything returned with just `kubectl get pods -n velero -l component=minio`?

Comment (carlisia) [reply docs/get-started.md]: Ahhh sorry, I didn't pay attention to that. Carry on! (I thought it was fetching all pods.)

**Pull request merged**